src/job/job.spec.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env jest */
+import config from "../_config";
+import { waitObjectState, xo } from "../util";
+
+describe("job", () => {
+  let defaultJob;
+
+  beforeAll(() => {
+    defaultJob = {
+      name: "jobTest",
+      timeout: 2000,
+      type: "call",
+      key: "snapshot",
+      method: "vm.snapshot",
+      paramsVector: {
+        type: "crossProduct",
+        items: [
+          {
+            type: "set",
+            values: [
+              {
+                id: config.vmIdXoTest,
+                name: "test-snapshot",
+              },
+            ],
+          },
+        ],
+      },
+    };
```

Comment:
Why in `beforeAll`? Why not directly assign `const defaultJob`?

Reply:
Because we need to wait the load of the configuration to get the `vmId`.